app/src/app/components/pretty-json/pretty-json.component.ts

```diff
@@ -43,6 +44,7 @@ export class PrettyJsonComponent implements OnInit {
   private listener      = null;
   editJson      = "";
   editorVisible = false;
+  readOnly      = environment.readOnly;
```

Comment:
this seems to override the env var `MONGOKU_READ_ONLY_MODE` (also it's not consistent w/ the middleware)

Reply:
Yes, for now it works with docker but not in the general use-case. At least not without tweaking the file manually.
I wanted to avoid having to make a request to get the value, but it might be easier

Reply:
Seems to be working great, thanks!